tools/searchSync.js

```diff
@@ -6,18 +6,17 @@ import { Pub, Page } from '../server/models';
 import { deletePubSearchData } from '../workers/tasks/search';
```

Comment:
Oops — I am messing around with this file locally (and actually running it right now, so I am a little afraid to revert these changes) but this really isn't part of the proposed changes.